project.code-workspace

```diff
@@ -8,7 +8,7 @@
   "settings": {
```

Comment:
Can we keep the `project.code-workspace` as is and not commit any changes to it, please?

Reply:
Yes, this was unintentional.